indico/modules/users/templates/personal_data.html

```diff
@@ -15,11 +15,63 @@
+                                {% set dialog_title = _('Delete') + ' {first_name} {last_name}?'.format(first_name=user.first_name, last_name=user.last_name) %}
+                            {% endif %}
+                            <ind-delete-button dialog-title="{{ dialog_title }}"
+                                               button-label="{% trans %}Delete User{% endtrans %}"
+                                               is-disabled="{{ user.is_admin | tojson }}"
+                                               disabled-message="{{ disabled_message }}"
+                                               message="{% trans %}Deleted user accounts cannot be restored.{% endtrans %}"
+                                               cancel-text="{% trans %}No, I don't{% endtrans %}"
+                                               confirm-text="{% trans %}Yes, I want to delete{% endtrans %} {{ user.first_name }}?"
+                                               record-name="{{ user.first_name }} {{ user.last_name }}"
+                                               delete-url="{{ url_for('users.user_delete', user) }}"
+                                               redirect-url="{{ url_for('users.users_admin') }}">
+                                {% trans %}Once deleted, the following will happen:{% endtrans %}
+                                <div style="padding-top: 15px; padding-bottom: 15px">
+                                    <ul class="ui list" style="list-style: none; padding-left: 0; margin-top: 0">
+                                        <li class="item">
+                                            <i class="minus circle icon"></i>
+                                            <div class="content">
+                                                <span>
+                                                    {% trans first_name=user.first_name %}
+                                                        <strong>{{ first_name }}</strong> will no longer be able to access Indico.
+                                                    {% endtrans %}
+                                                </span>
+                                            </div>
+                                        </li>
+                                        <li class="item">
+                                            <i class="times circle outline icon"></i>
+                                            <div class="content">
+                                                <span>
+                                                    {% trans first_name=user.first_name %}
+                                                        <strong>{{ first_name }}</strong> will be removed from all areas of Indico - this does not affect
+                                                        their presence in events as a speaker or other role; they will still be listed in such
+                                                        capacities where applicable.
```

Comment:
I think it would be positive to more clearly distinguish between Indico user accounts and associated personal data. What does it mean "all areas of Indico" when the next sentence explicitly states the opposite? Not completely thought out but I believe that this would be closer to reality:

> <strong>{{ first_name }}</strong>'s user account will be permanently deleted from Indico - this does not remove their personal information from events as participant, speaker or other role; they will still be listed in such capacities where applicable.

/cc @ThiefMaster

Reply:
I think this is out of scope for this PR, since it just moves things around?

Reply:
I know but it's the one thing that stood out :)